src/main/java/no/nav/pto/veilarbfilter/domene/NyOverblikkVisningModel.java

```diff
@@ -0,0 +1,20 @@
+package no.nav.pto.veilarbfilter.domene;
+
+import com.fasterxml.jackson.annotation.JsonSetter;
+import com.fasterxml.jackson.annotation.Nulls;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import lombok.*;
+import org.flywaydb.core.internal.configuration.ListDeserializer;
+
+import java.util.List;
+
+@Data
+@Getter
+@Setter
+@AllArgsConstructor
+@JsonDeserialize
+@NoArgsConstructor
+public class NyOverblikkVisningModel {
+    @JsonSetter(nulls = Nulls.AS_EMPTY)
+    List<String> overblikkVisning;
+}
```

Comment:
Flere av disse annotasjonene er Lombok-annotasjoner som genererer kode som vi ikke trenger. Vi kan fjerne de og heller bruke Java sin innebygde [`record`-funksjonalitet](https://docs.oracle.com/en/java/javase/20/language/records.html#GUID-6699E26F-4A9B-4393-A08B-1E47D4B2D263):

```suggestion
@JsonDeserialize
public record NyOverblikkVisningModel(
    @JsonSetter(nulls = Nulls.AS_EMPTY)
    List<String> overblikkVisning;
) {
}
```